crates/core/c8y_smartrest/src/topic.rs

```diff
@@ -93,6 +100,32 @@ impl TryFrom<Topic> for MapperSubscribeTopic {
     }
 }
 
+/// returns a c8y message specifying to set log status to executing.
+///
+/// example message: '501,c8y_LogfileRequest'
+pub async fn get_log_file_request_executing() -> Result<Message, SMCumulocityMapperError> {
```

Comment:
Do these 2 new methods really belong in this `topic.rs` file? The original file was more generic and now we're overloading it with operation specific logic. Wondering if these methods can be moved to the `log_request_plugin/src/smartrest.rs` which contains the log-request specific smart-rest logic? You might wanna add some unit tests as well for these methods as well, which look pretty straightforward.